.github/how_to_compile_gudhi_in_a_conda_env.md

```diff
@@ -0,0 +1,97 @@
+# Install a conda development environment to compile GUDHI
+
+## Install miniconda
+
+Download the [installer](https://docs.conda.io/en/latest/miniconda.html) required by your system and follow the [instructions](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
+
+## Create a dedicated environment
+
+```bash
+conda install -c conda-forge mamba  # installation with mamba is faster
+conda create --name gudhi
+conda activate gudhi
+mamba install -c conda-forge python cmake doxygen eigen cgal-cpp
+```
+
+Some of the requirements are in the gudhi repository (please refer to
+[how to use github to contribute to gudhi](how_to_use_github_to_contribute_to_gudhi.md)).
+In the gudhi repository - let's call it `/workdir/gudhi` i.e. - once submodules are initialised:
+
+```bash
+pip install -r ext/gudhi-deploy/build-requirements.txt 
+pip install -r ext/gudhi-deploy/test-requirements.txt  # pytorch can be painful to install - not mandatory
+```
+
+## Compilation
+
+In order to compile all c++ utilities, examples, benchmarks, unitary tests, and python module:
+```bash
+cd /workdir/gudhi
+rm -rf build
```

Comment:
Maybe add a comment here explaining that any existing 'build' folder will be removed (in case the commands are just run by the user without really paying attention to what they are doing).